index.html

```diff
@@ -0,0 +1,357 @@
+                    <p class="activity__text">op funda sinds:<span class="">24</span></p>
+                </div>
+
+            </div>
+
+            <div>
+                <h2>buurt</h2>
+
+                <table>
+                    <thead>
+                    <tr>
+                        <th>waterrijk</th>
+                        <th></th> <!-- Added an empty column -->
+                    </tr>
+                    </thead>
+                    <tbody>
+                    <tr>
+                        <td>inwoners</td>
+                        <td>w5610</td>
+                    </tr>
+                    <tr>
+                        <td>Gezin met kinderen</td>
+                        <td>55%</td>
+                    </tr>
+                    <tr>
+                        <td>Gem. vraagprijs / m²</td>
+                        <td>€ 3.692</td>
+                    </tr>
+
+                    </tbody>
+                </table>
+                <button type="button" class="button-show-charactize">meer over</button>
+
+            </div>
+
+            <div class="in-de-buurt"><h2>In de buurt</h2>
+                <ul>
+                    <li>
+                        <a href="">basischool</a></li>
+                    <li><a href="">supermarkt</a></li>
+                    <li><a href="">bushalter</a>
+                    </li>
+                </ul>
+
+            </div>
+            <div>
+                <img alt="map-images" class="image-change" src="images/map.png">
+            </div>
+
+        </div>
+        <div class="gridconainer__small-images-div">
+            <ul class="gridcontainer__small-images">
+
+
+                <li class="carousel-images__no-dot">
+                    <img alt="hoofdfoto huis" class="gridcontainer__small-images__images-next-to-main-image images-houses-2"
+                         src="images/eettafel.jpg">
+                </li>
+                <li class="carousel-images__no-dot"> <img alt="hoofdfoto huis" class="gridcontainer__small-images__images-next-to-main-image images-houses-3" src="images/woonkamer.jpg">   </li>
+                <li class="carousel-images__no-dot"> <img alt="hoofdfoto huis" class="gridcontainer__small-images__images-next-to-main-image images-houses-4" src="images/tuin.jpg">   </li>
+                <li class="carousel-images__no-dot"> <img alt="hoofdfoto huis" class="gridcontainer__small-images__images-next-to-main-image images-houses-5" src="images/video.jpg">   </li>
+                <li class="carousel-images__no-dot"> <img alt="hoofdfoto huis" class="gridcontainer__small-images__images-next-to-main-image images-houses-6" src="images/stairs.jpg">   </li>
+                <li class="carousel-images__no-dot"> <img alt="hoofdfoto huis" class="gridcontainer__small-images__images-next-to-main-image images-houses-7" src="images/street.jpg">   </li>
+            </ul>
+        </div>
```

Comment:
Very nice to see you use a [CSS methodology / naming convention](https://getbem.com/introduction/) to structure your classes and make it easier to target them. However, consider reducing your classes as much as possible, because you're working in a single HTML File. It can quickly look bloated and makes it difficult to read.

What are the reasons you decided for this approach? Feel free to ignore this code suggestion, becomes it comes down to "style". Or maybe you can find the right balance between using a CSS methodology and reducing the amount of classes.

The `<li>` element suggests this is a carousel given the class `carousel-images`, but if you're going to use a CSS methodology, consider giving the parent/root element of the feature a descriptive class name f.e. `carousel` and then target the children with specific names f.e. `carousel-image`. Feel free to read more on the link above about using CSS methodologies if you plan on sticking with this approach.

```suggestion
        <div class="main-slider">
            <ul>
                <li><img alt="hoofdfoto huis" src="images/eettafel.jpg"></li>
                <li><img alt="hoofdfoto woonkamer" src="images/woonkamer.jpg"></li>
                <li><img alt="hoofdfoto tuin" src="images/tuin.jpg"></li>
                <li><img alt="hoofdfoto video" src="images/video.jpg"></li>
                <li><img alt="hoofdfoto trap" src="images/stairs.jpg"></li>
                <li><img alt="hoofdfoto straat" src="images/street.jpg"></li>
            </ul>
        </div>
```